ArchiSteamFarm/Core/Utilities.cs

```diff
@@ -327,7 +329,7 @@ public static class Utilities {
 			}
 		}
 
-		internal static bool IsWeakPassword(string password, ISet<string>? additionallyForbiddenPhrases = null) {
+		internal static bool IsWeakPassword(string password, out string? reason, ISet<string>? additionallyForbiddenPhrases = null) {
```

Comment:
I hate mixing `out`, nullable and everything else together. Just make it return `(bool Result, string? Reason)`. Or change name of the method to `TestPasswordStrength` and then `(bool IsWeak, string? Reason)`.